platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/SolrCloudClientFactory.java

```diff
@@ -49,127 +49,92 @@
  * </ul>
  */
 public class SolrCloudClientFactory implements SolrClientFactory {
-
   private static final Logger LOGGER = LoggerFactory.getLogger(SolrCloudClientFactory.class);
 
-  private static final int SHARD_COUNT =
-      NumberUtils.toInt(System.getProperty("solr.cloud.shardCount"), 2);
+  private final int shardCount = NumberUtils.toInt(System.getProperty("solr.cloud.shardCount"), 2);
 
-  private static final int REPLICATION_FACTOR =
+  private final int replicationFactor =
```

Comment:
Yes, because they are referencing system properties, it makes it easier to test as we can change values in between tests without resorting to making these non-final and defining setters.